### DISCUSSION TITLE
How to edit my profile page on github

### DISCUSSION THREAD

**Discussion by CodeLK4** [community/community#65091] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

what are the languages that i can use

**Comment by LinukaAr** [comment]
Open your profile by clicking on your profile picture or username.
Look for and click on the "Edit profile" button.
Make desired changes to your details like profile picture, bio, etc.
Save your changes using the appropriate button.
 You can use html/css/java/python 
 programming languages you can use on GitHub are not limited to this list. The choice of language depends on the type of project you're working on and your personal preferences. GitHub supports a diverse range of programming languages, making it a hub for developers from different backgrounds and specialties.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
